management-gateway/src/test/java/org/frankframework/management/bus/BusResponseTypesTest.java

```diff
@@ -72,7 +72,7 @@ public void testJsonStructure() {
 		json.add("key", "value");
 		JsonMessage message = new JsonMessage(json.build());
 
-		assertEquals("{\"key\":\"value\"}", message.getPayload().replaceAll("\s", "").replaceAll("\n", ""));
+		assertEquals("{\"key\":\"value\"}", message.getPayload().replaceAll(" ", "").replaceAll("\n", ""));
```

Comment:
Is dit handig?

Reply:
Zekers. Is direct duidelijk en werkt hetzelfde.

Reply:
Weet je het zeker?

`\s	A whitespace character: [ \t\n\x0B\f\r]`

Nu replace je alleen spaties volgens mij.

Reply:
Ik had het zojuist opgezocht, maar blijkbaar klopt die definitie die ik vond niet? Ik zet het wel terug.